code/modules/clothing/masks/gasmask.dm

```diff
@@ -64,7 +123,20 @@ GLOBAL_LIST_INIT(clown_mask_options, list(
 	has_filter = TRUE
 	return TRUE
 
+/obj/item/clothing/mask/gas/equipped(mob/M, slot)
+	. = ..()
+	if(cig)
+		cig.equipped(M, slot)
```

Comment:
You have `/obj/item/clothing/mask/gas/equipped` implemented twice, which means you're duping calling `cig.equipped`